.github/workflows/trigger.yml

```diff
@@ -0,0 +1,34 @@
+name: Tag & Release Binaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: Version number of the release
+        default: 0.5.0
```

Comment:
Is there even a sensible default that we can use here?

Reply:
Maybe it is possible to extract this from signer here?